aspen/http/resource.py

```diff
@@ -0,0 +1,75 @@
+from .response import Response
+from ..simplates import Simplate, SimplateDefaults, SimplateException
+
+
+class Static(object):
+    """Model a static HTTP resource.
```

Comment:
Yea, but this is too abstract. Is it an object that represents static resource on disk, in template, or in web request?